index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Hello Microverse</title>
+</head>
+<body>
+    <header>
+        <h1>Hello Microverse!</h1>
```

Comment:
- [ ] Add a class to this element so that you can use it to highlight and apply styles in the CSS file.

Reply:
Okay, let me do it right!